sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -328,18 +328,19 @@ class SQLContext(@transient val sparkContext: SparkContext)
     def numPartitions = self.numShufflePartitions
 
     def strategies: Seq[Strategy] =
-      extraStrategies ++ (
```

Comment:
As @chenghao-intel said, do you mind not changing this part? It would be great if the PR focuses on what it does.